packages/atomic/src/components/common/facets/category-facet/child-value-link.ts

```diff
@@ -0,0 +1,18 @@
+import type {FunctionalComponentWithChildren} from '@/src/utils/functional-component-utils';
+import {
+  renderCategoryFacetValueLink,
+  type CategoryFacetValueLinkProps,
+} from './value-link';
+
+interface CategoryFacetChildValueLinkProps
+  extends Omit<CategoryFacetValueLinkProps, 'isParent'> {}
+
+export const renderCategoryFacetChildValueLink: FunctionalComponentWithChildren<
```

Comment:
I don't think this should exist as a functional component; we should simply call `renderCateogryFacetValueLink` with `isParent: false` when we need this behavior.